recipes/django-elasticsearch-dsl/LICENSE

```diff
@@ -0,0 +1,15 @@
+Apache Software License 2.0
+
+Copyright (c) 2016, Sabre
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Looks like the license is included in the PyPI tarball, so you should not include this file.